pkg/execution/state/redis_state/lua/queue/dequeue.lua

```diff
@@ -13,9 +13,10 @@ local idempotencyKey = KEYS[4]
 -- We must dequeue our queue item ID from each concurrency queue
 local accountConcurrencyKey   = KEYS[5] -- Account concurrency level
 local partitionConcurrencyKey = KEYS[6] -- Partition (function) concurrency level
-local customConcurrencyKey    = KEYS[7] -- Optional for eg. for concurrency amongst steps 
+local customConcurrencyKeyA   = KEYS[7] -- Optional for eg. for concurrency amongst steps 
+local customConcurrencyKeyB   = KEYS[8] -- Optional for eg. for concurrency amongst steps 
```

Comment:
Eh.  It annoys me but we'll move to FDB and be done with it at some point.  For Redis, you need to explicitly list all keys so that it can check for CROSSSLOT errors.  You cannot query across shards in Redis.